applications/ParticleMechanicsApplication/tests/test_particle_vtk_output_process.py

```diff
@@ -0,0 +1,215 @@
+import KratosMultiphysics
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import KratosMultiphysics.kratos_utilities as kratos_utils
+import KratosMultiphysics.ParticleMechanicsApplication.particle_vtk_output_process as particle_vtk_output_process
+from KratosMultiphysics.compare_two_files_check_process import CompareTwoFilesCheckProcess
+
+import os
+import pathlib
+
+class TestParticleVtkOutputProcess(KratosUnittest.TestCase):
+    def test_ascii_particle_conditions_vtk_output_2D(self):
+        ExecuteBasicParticleVTKOutputProcessCheck("ascii", "condition")
+
+    def test_ascii_particle_elements_vtk_output_2D(self):
+        ExecuteBasicParticleVTKOutputProcessCheck("ascii", "element")
+
+    def test_binary_particle_conditions_vtk_output_2D(self):
+        ExecuteBasicParticleVTKOutputProcessCheck("binary", "condition")
```

Comment:
```suggestion
    def test_binary_particle_conditions_vtk_output_2D(self):
        if os.system == "nt:
            self.skipTest("Binary output currently not working in windows.")
        ExecuteBasicParticleVTKOutputProcessCheck("binary", "condition")
```

this is the proper way of skipping tests, this way it gets reported correctly by the test runner

Reply:
Ok, thanks for the suggestion